static/css/styles.css

```diff
@@ -400,12 +420,18 @@ select { /* https://stackoverflow.com/questions/3532649/problem-with-select-and-
     appearance: none;
 }
 
-label[for=field-language] {
+label[for=field-language],
+label[for=media-type],
+label[for=format],
+label[for=section] {
```

Comment:
Is there no `class=select` on the wrapper that you do each manually?

Reply:
`select` is inside `label` :-(